eclipse-scout-core/test/form/fields/FormFieldSpec.js

```diff
@@ -393,6 +393,39 @@ describe('FormField', () => {
     expect(formField._isSuppressStatusIcon()).toBe(false);
   });
 
+  describe('property errorStatus', () => {
+    let formField, model;
+
+    beforeEach(() => {
+      model = helper.createFieldModel();
+      formField = createFormField(model);
+    });
+
+    it('added by addErrorStatus with Status and String as argument', () => {
+      expect(formField.errorStatus).toEqual(null);
+
+      formField.addErrorStatus(Status.warning('warning'));
+
```

Comment:
There are still unnecessary blank lines between the setup and the assertion ;-)

-> remove blank lines before expect()